starter-code/javascript/main.js

```diff
@@ -51,10 +59,29 @@ function setResetBtn() {
 
 // Start/Stop Button
 btnLeft.addEventListener('click', function () {
-
+    if (btnLeft.innerText === "START") {
+        btnLeft.className = "btn stop";
+        btnLeft.textContent = "STOP";
+        btnRight.className = "btn split";
+        btnRight.textContent = "SPLIT";
+        chronometer.startClick();
```

Comment:
Here you started the chronometer, but you should also call your function printTime(), that will continuously update the DOM with a setInterval.